requirements.in

```diff
@@ -46,4 +46,6 @@ opentelemetry-instrumentation-celery>=0.45b0
 opentelemetry-sdk>=1.24.0
 django-admin
 google-cloud-pubsub
-openai
\ No newline at end of file
+openai
+zipp>=3.19.1
```

Comment:
Please try to update `opentelemetry-instrumentation-celery` (has a new version 0.46b0) and / or `opentelemetry-sdk` (has a new version 1.25.0) to see if that fixes the zipp issue

Reply:
Yeah I tested this and it's not enough. `zipp` won't automatically upgrade if you update any of those 2.

However if you run the command below it does upgrade in the `requirements.txt` file without having to explicitly require it :E
```
$ pip-compile requirements.in --upgrade-package zipp
```

Reply:
i did not know about that command, very useful for this kind of task